scripts/e2e-compose-synchronous-write.sh

```
@@ -0,0 +1,63 @@
+NODE
+}
+
+run_synchronous_write_e2e() {
+  local team_id="$1"
+  local slice="${DENSE_MEM_E2E_WRITE_SLICE:-legacy}"
+  echo "Running compose-backed synchronous-write contract cases with slice ${slice}."
+  DENSE_MEM_USER_URL="$USER_URL" \
+  DENSE_MEM_CONTROL_URL="$CONTROL_URL" \
+  DENSE_MEM_CONTROL_TOKEN="$CONTROL_TOKEN" \
+  DENSE_MEM_E2E_TEAM_ID="$team_id" \
+  DENSE_MEM_E2E_API_KEY="$api_key" \
```

Shellcheck (0.11.0)

[warning] api_key is referenced but not assigned.

(SC2154)